docs/_howto/stub-dependency.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: How to stub a dependency of a module?
+---
+
+To stub dependency(imported module) of a module under test you have to import it explicitly in your test and stub the desired method.
+
+## Example
+
+```javascript
+import * as importedModule from "./importedModule";
+import {getTheSecret} from "./moduleUnderTest";
+
+"test should get secret": function () {
```

Comment:
This isn't actually valid javascript as it is and will result in a runtime error (`Unexpected token, expected ;`). Could you change it to the standard BDD style Mocha test style we use everywhere else?